internal/target/service_list_more.go

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package target
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/refreshtoken"
+	rt "github.com/hashicorp/boundary/internal/refreshtoken"
+)
+
+func ListMore(
```

Comment:
This name isn't final, but we'd want something to illustrate that this is different from the default list because it takes a refresh token, and as such needs to get the deleted IDs.

Reply:
Maybe something like `RefreshList`, `ListRefresh`, or `ListUpdatedSince`?